pkg/registry/proto/registry.proto

```diff
@@ -59,6 +59,16 @@ service AccountsService {
             delete: "/accounts/{uuid}"
         };
     };
+    rpc Suspend(nocloud.registry.accounts.SuspendRequest) returns (nocloud.registry.accounts.SuspendResponse) {
+        option (google.api.http) = {
+            get: "/accounts/{uuid}/suspend"
+        };
+    };
+    rpc Unsuspend(nocloud.registry.accounts.UnsuspendRequest) returns (nocloud.registry.accounts.UnsuspendResponse) {
+        option (google.api.http) = {
+            get: "/accounts/{uuid}/unsuspend"
+        };
+    };
```

Comment:
Why use `get` method even though we are already using post in modules?
https://github.com/slntopp/nocloud/blob/0f3bef2666c09c654f04f79c519f895d2241c417/pkg/services/proto/services.proto#L170-#L192